ci-operator/step-registry/aks/provision/aks-provision-commands.sh

```diff
@@ -3,13 +3,29 @@
 set -euo pipefail
 
 AZURE_AUTH_LOCATION="${CLUSTER_PROFILE_DIR}/osServicePrincipal.json"
+if [[ "${USE_HYPERSHIFT_AZURE_CREDS}" == "true" ]]; then
+    AZURE_AUTH_LOCATION="/etc/hypershift-ci-jobs-azurecreds/credentials.json"
+fi
 AZURE_AUTH_CLIENT_ID="$(<"${AZURE_AUTH_LOCATION}" jq -r .clientId)"
 AZURE_AUTH_CLIENT_SECRET="$(<"${AZURE_AUTH_LOCATION}" jq -r .clientSecret)"
 AZURE_AUTH_TENANT_ID="$(<"${AZURE_AUTH_LOCATION}" jq -r .tenantId)"
-AZURE_LOCATION="$LEASED_RESOURCE"
+AZURE_LOCATION="${HYPERSHIFT_AZURE_LOCATION:-${LEASED_RESOURCE}}"
```

Comment:
It might be helpful to define quota slices for the `hypershift` clusterprofile in the `{region}--hypershift-quota-slice-{idx}` similar to the following:
https://github.com/openshift/release/blob/a2003bca867eaafadf2fc33e6ddee45b41d99de0/core-services/prow/02_config/_boskos.yaml#L725-L757

So we could leverage `$LEASED_RESOURCE` without having to pass an extra environment variable. 

But for now I guess we could keep the current implementation.

Reply:
Hmmm we dont use these anywhere within the hypershift e2e afaik.  @enxebre is this something we would like to explore if so I can look into it.